spec/features/admin/add_custom_organisation_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'support/confirmation_use_case'
+
+describe 'adding a custom organisation name' do
+  include_examples 'confirmation use case spy'
+
+  let!(:admin_user) { create(:user, super_admin: true) }
+
+  context 'when visiting the custom organisations page' do
+    before do
+      sign_in_user admin_user
+      visit admin_custom_organisations_path
+    end
+
+    it 'will show the add custom organisations page' do
+      expect(page.body).to have_content('Allow an organisation access to the admin platform')
+    end
+
+    it 'will redirect you to organiastions page once a custom org is added' do
+      fill_in "Enter the organisation's full name", with: 'Custom Org name'
+      click_on 'Allow organisation'
+      expect(page).to have_content('Custom organisation has been successfully added')
+    end
+
+    context 'sign out and find custom organisation' do
+      before do
+        fill_in "Enter the organisation's full name", with: 'Custom Org name'
+        click_on 'Allow organisation'
+        sign_out
+        sign_up_for_account(email: 'default@gov.uk')
+        visit confirmation_email_link
+      end
+
+      it 'the custom organisation will appear when creating an account' do
```

Comment:
`it 'the` doesn't make sense. Specs need to be readable. Something like `it 'displays the custom organisation name in the list` would be better for instance.